### ISSUE TITLE
Add graph algorithms

### ISSUE THREAD

**Issue by rohanprasadofficial** [rohanpd4/Algorithms#1]

**Comment by NidhiNivedita6** [comment]
Can I add Newton Raphson Method in C?

**Comment by Mayankjha997** [comment]
Hello sir!
I would like to add lagrange interpolation method in c++ language

**Comment by rohanprasadofficial** [comment]
Yes go ahead create a new folder as "Graph Algorithms"  and then make a PR, make sure that you write clean commented code 
@NidhiNivedita6  @Mayankjha997

**Comment by Mayankjha997** [comment]
@rohanprasadofficial is this contribution will add up in hacktoberfest contribution.

**Comment by predator4hack** [comment]
Can I add the Union-Find Algorithm?

**Comment by pbuggaveeti28** [comment]
Can I add Divide and Conquer algorithms?

**Comment by rohanprasadofficial** [comment]
Hey Everyone, you all write clean code and do a PR I will review them once tonight and approve them if its a good once ?

Make sure that you don't add duplicate request. 

Remember to add code in python as it is a python codebase.
For C++ checkout : 
https://github.com/rohanprasadofficial/Data-Structures-Using-CPP

**Comment by Mayankjha997** [comment]
@rohanprasadofficial please merge my PR #3
